### DISCUSSION TITLE
Function of some arguments

### DISCUSSION THREAD

**Discussion by kareefardi** [The-OpenROAD-Project/OpenROAD#4315] [Q&A] [answered]
I am asking about the function of these arguments. There is no documentation for them:
```
-skip_pin_swap
-skip_gate_cloning
-floorplan_initialize
```

**Comment by maliberty** [comment, marked as answer]
It is helpful to say what they are arguments of.  

read_def -floorplan_initialize :   does not read the netlist but just floorplan information like pin locations, instance locations, etc.
repair_timing -skip_pin_swap & -skip_gate_cloning :  disable two new optimizations in case of trouble.  More of an escape than something we expect people to use regularly.

**Reply by kareefardi** [reply]
Thank you. Will do so in the future.
